package.json

```diff
@@ -37,7 +37,7 @@
     "prepublish": "tsc",
     "example": "yarn --cwd examples/purchaseTester",
     "pods": "cd examples/purchaseTester && pod-install --quiet",
-    "bootstrap": "yarn examples/purchaseTester && yarn && yarn pods"
```

Comment:
I actually fixed this here too https://github.com/RevenueCat/react-native-purchases/pull/234/files/b753dca47b5c256ccaab79dc9bc2679fe9e8e751#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L40

The script example already calls yarn in the folder:

```
    "example": "yarn --cwd examples/purchaseTester",
```

So I think this can be rolled back to yarn example (the change was introduced here https://github.com/RevenueCat/react-native-purchases/pull/232/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L40)